### DISCUSSION TITLE
How to get a developer license with no risk of leaking PII

### DISCUSSION THREAD

**Discussion by sandrock** [ravendb/ravendb#17133] [General]
I would like to obtain a developer license without sending data to Google. And I believe not only me has this wish. 

[This form](https://ravendb.net/license/request/dev) on the website requires running google recaptcha scripts. I tried and observed a HTTP POST request sending 6710 bytes of obfuscated data. 

How can be be sure the form data has not been sent?

Would you consider changing your captcha provider for a less suspect entity?

---

edit: I was able to de-obfuscate some parts of the payload, but most of it remains unreadble.

**Comment by ppekrol** [comment]
Hi, please write directly to sales@ravendb.net and our team will handle your request for a Dev license.
